getnacelle/nacelle-js
# feat: automate changelog sync to readme docs

## Why are these changes introduced?

Addresses [ENG-8894](https://nacelle.atlassian.net/browse/ENG-8894) 



## What is this pull request doing?



- Add new readme file generator module that adds a front matter to the` CHANGELOG.md` and creates a new file with the new string
- Add command to SDK to call that module
- Add GitHub actions that automates the process of generating the new `readme` file and syncs it to the Nacelle Changelogs docs
- Trigger GitHub release workflow on pushes to `ENG-8736/rebase-to-prepare-for-2.0.0`


## How to Test



### With `rdme`
To test the `rdme` sync:
- [Install `rdme`](https://github.com/readmeio/rdme#installing-rdme-to-your-local-machine)
- run `rdme changelogs packages/storefront-sdk/changelog-readme.md --dryRun` at the root

For the action:
- Merging to `ENG-8736/rebase-to-prepare-for-2.0.0` will ensure it works 

## Checklist



[ENG-8894]: https://nacelle.atlassian.net/browse/ENG-8894?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
